tests/docker-compose.yml

```diff
@@ -1,33 +1,58 @@
-version: '2.1'
+version: "2.1"
 services:
-  apm-server:
-    depends_on:
-      - elasticsearch
-      - kibana
-      - agent-python
-    build: .
-    ports:
-      - "8200:8200"
-    command: ./apm-server -E output.elasticsearch.hosts=[http://elastic:changeme@elasticsearch:9200] -E apm-server.host=0.0.0.0:8200 -e
-
   elasticsearch:
-    extends:
-      file: ../_beats/testing/environments/snapshot.yml
-      service: elasticsearch
+    image: docker.elastic.co/elasticsearch/elasticsearch-platinum:${STACK_VERSION:-6.1.2}
```

Comment:
minor thing: I'd remove the default and just point out in the docs to set the version explicitly like `STACK_VERSION=version make start-dev`, that way we don't keep an old default when new versions roll out and we don't have to remember to update it.